client/src/components/TimePreferencesModal.js

```diff
@@ -208,6 +207,7 @@ export default class TimePreferencesModal extends Component {
                             icon: "success",
                         }).then(() => {
                             this.props.closeModal();
+                            window.location.reload();
```

Comment:
Added a window.location.reload whenever new topcis and times are successfully added, to cause a refetch of profile data with updated time/topic preferences. Ideally, we can refactor this to not do a refresh, let me know what you think.

Reply:
I'm assuming that this is intended to refresh the parent component (AlumniProfile)'s profile data from the backend, allowing the modal to open with updated props

This gives us a couple of options, because we have to make a call to the backend either way
- We can switch from giving availabilities as a prop to letting the modal get the availabilities from the backend using id passed as a prop. This gives extra independence to the preferences modal but means the profile page won't always reflect the changes. This can be done with a ComponentWillMount because the Profile will unmount the component upon submission
- We can set fetch the profile's data in this.closeTimePreferencesModal in AlumniProfile before setting the state of the modal to closed. This means that it is updated immediately upon  submission, and if we ever decide to display preferences on the profile we could update the page. In terms of work, this is probably a lot cheaper and will be a pattern we can reuse later for updating the profile in general

Reply:
I really like the second option! Good call!